c#/Mandoline.Api.Examples/Client.Repl/InfoCommand.cs

```diff
@@ -6,16 +6,10 @@
 
 namespace Client.Repl
 {
-    using System;
-    using System.Collections.Generic;
-    using System.Linq;
-    using System.Text;
     using System.Threading.Tasks;
     using Core;
-    using Mandoline.Api.Client;
-    using Replify;
 
-    public class InfoCommand : IReplCommand
+    public class InfoCommand
```

Comment:
is there a reason why we are not implementing the IReplCommand interface any more?

Reply:
IReplCommand was used to implement commands when we were using the Replify library I believe, which we have now replaced with Sharprompt. We now implement the commands in the Program.cs class directly.

Reply:
Thanks @ryank645, yeah wasn't sure about Sharprompt